azurelinuxagent/ga/telemetry_service.py

```diff
@@ -38,14 +39,17 @@ class TelemetryServiceHandler(ThreadHandlerInterface):
     """
 
     _THREAD_NAME = "TelemetryServiceHandler"
-    _MAX_TIMEOUT = datetime.timedelta(minutes=5).seconds
+    _MAX_TIMEOUT = datetime.timedelta(seconds=5).seconds
+    _MIN_QUEUE_LIMIT = 30
```

Comment:
Minor suggestion: rename this for the usage, not the implementation, e.g. `_MIN_EVENT_LIMIT`.

Also, the naming `MIN` is confusing since you want it to be `< MIN`. Maybe `_MAX_EVENTS_TO_BATCH`?

Reply:
Ahh makes sense. I named it `MIN` because this is the min number of events we will wait for, `MAX` seems counterintuitive. I'll rename it to something like `_MIN_EVENT_TO_BATCH` as it makes more sense given the use case.

Reply:
Done